LongWarOfTheChosen/Src/LW_Overhaul/Classes/UIMission_LWLaunchDelayedMission.uc

```diff
@@ -52,6 +57,138 @@ simulated function Name GetLibraryID()
+			*     ShadowChamber = Spawn(class'UIAlertShadowChamberPanel', LibraryPanel);
+			*     ShadowChamber.InitPanel('UIAlertShadowChamberPanel', 'Alert_ShadowChamber');
+			*
+			* This is done so that normal missionss (gops, suply raids etc..) do not show shadow chamber information as
+			* this behavior is what vanilla LW2 implemented.
+		    **********************************************************************************/
+			AlertLibID = GetLibraryID();
+
+			if( AlertLibID != '' )
+			{
+				LibraryPanel = Spawn(class'UIPanel', self);
+				LibraryPanel.bAnimateOnInit = false;
+				LibraryPanel.InitPanel('LibraryPanel', AlertLibID);
+		
+				ButtonGroup = Spawn(class'UIPanel', LibraryPanel);
+				ButtonGroup.InitPanel('ButtonGroup', '');
+		
+				Button1 = Spawn(class'UIButton', ButtonGroup);
+				Button2 = Spawn(class'UIButton', ButtonGroup);
+		
+				Button3 = Spawn(class'UIButton', ButtonGroup);
+				Button3.SetResizeToText( false );
+				Button3.InitButton('Button2', "",, eUIButtonStyle_NONE);
+				Button3.OnSizeRealized = OnButtonSizeRealized;
+		
+				ConfirmButton = Spawn(class'UIButton', LibraryPanel);
+		
+				if( `ISCONTROLLERACTIVE == false )
+				{
+					ConfirmButton.InitButton('ConfirmButton', "", OnLaunchClicked);
+					Button1.InitButton('Button0', "",, eUIButtonStyle_NONE);
+					Button2.InitButton('Button1', "",, eUIButtonStyle_NONE);
+					ConfirmButton.SetResizeToText(false);
+					Button1.SetResizeToText( false );
+					Button2.SetResizeToText(false);
+					Button1.OnSizeRealized = OnButtonSizeRealized;
+					Button2.OnSizeRealized = OnButtonSizeRealized;
+				}
+				else
+				{
+					//bsg-jneal (5.11.17): fix buttons styles on UIMissions
+					Button1.InitButton('Button0', "", OnLaunchClicked, eUIButtonStyle_NONE);
+					Button1.SetGamepadIcon(class'UIUtilities_Input'.static.GetAdvanceButtonIcon());
+					Button1.OnSizeRealized = OnButtonSizeRealized;
+					Button1.SetResizeToText(false);
+					
+					Button2.InitButton('Button1', "", OnCancelClicked, eUIButtonStyle_NONE);
+					Button2.SetGamepadIcon(class'UIUtilities_Input'.static.GetBackButtonIcon());
+					Button2.OnSizeRealized = OnButtonSizeRealized;
+					Button2.SetResizeToText(false);
+					//bsg-jneal (5.11.17): end
+		
+					ConfirmButton.InitButton('ConfirmButton', "", OnLaunchClicked, eUIButtonStyle_HOTLINK_WHEN_SANS_MOUSE);
+					ConfirmButton.SetGamepadIcon(class'UIUtilities_Input'.static.GetAdvanceButtonIcon());
+					ConfirmButton.OnSizeRealized = OnButtonSizeRealized;
+					ConfirmButton.SetX(1450.0);
+					ConfirmButton.SetY(617.0);
+					ConfirmButton.SetWidth(300);
+				}
+		
+				ConfirmButton.DisableNavigation();
+		
+				SitrepPanel = Spawn(class'UIAlertSitRepPanel', LibraryPanel);
+				SitrepPanel.InitPanel('SitRep', 'Alert_SitRep');
+				SitrepPanel.SetTitle(m_strSitrepTitle);
+		
+				ChosenPanel = Spawn(class'UIPanel', LibraryPanel).InitPanel(, 'Alert_ChosenRegionInfo');
+				ChosenPanel.DisableNavigation();
+			}
+
+		    break;
+	}
+}
+
```

Comment:
As with the earlier commit, can't we just call the super method and then hide the shadow chamber panel?

Reply:
resolved with my latest commit.

Reply:
Is the rest of the copied code necessary here, i.e. what was copied from `UIMission_LWCustomMission`? Couldn't the entire implementation of this method simply be:

```
super.BindLibraryItem();

ShadowChamber.Hide();
```